docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -301,54 +259,25 @@ Make a choice based on whether or not you already have a Prometheus server set u
 
      > **Note:** On Linux systems, if \<ingress-host\> cannot be resolved by the Prometheus server, use the additional command-line flag `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` to set it up.
 
-### Configure Grafana Agent to write to Grafana Mimir
-
-You can either configure Grafana Agent to write to Grafana Mimir or [configure Prometheus to write to Mimir](#configure-prometheus-to-write-to-grafana-mimir). Although you can configure both, you do not need to.
-
-Make a choice based on whether or not you already have a Grafana Agent set up:
-
-- For an existing Grafana Agent:
+### Configure Grafana Alloy to write to Grafana Mimir
 
-  1. Add the following YAML snippet to your Grafana Agent metrics configurations (`metrics.configs`):
-
-     ```yaml
-     remote_write:
-       - url: http://<ingress-host>/api/v1/push
-     ```
+You can either configure Grafana Alloy to write to Grafana Mimir or [configure Prometheus to write to Mimir](#configure-prometheus-to-write-to-grafana-mimir). Although you can configure both, you don't need to.
 
-     In this case, your Grafana Agent will write metrics to Grafana Mimir, based on what is defined in the existing `metrics.configs.scrape_configs` configuration.
+To configure Grafana Alloy to write to Mimir, use the `prometheus.remote_write` component. For example:
 
-  1. Restart the Grafana Agent.
+```
+prometheus.remote_write "LABEL" {
+  endpoint {
+    url = http://<ingress-host>/api/v1/push
 
-- For a Grafana Agent that does not exist yet:
+    ...
+  }
 
-  1. Write the following configuration to an `agent.yaml` file:
-
-     ```yaml
-     metrics:
-       wal_directory: /tmp/grafana-agent/wal
-
-       configs:
-         - name: agent
-           scrape_configs:
-             - job_name: agent
-               static_configs:
-                 - targets: ["127.0.0.1:12345"]
-           remote_write:
-             - url: http://<ingress-host>/api/v1/push
-     ```
-
-     In this case, your Grafana Agent writes metrics to Grafana Mimir that it scrapes from itself.
-
-  1. Create an empty directory for the write ahead log (WAL) of the Grafana Agent
-
-  1. Start a Grafana Agent by using Docker:
-
-     ```bash
-     docker run -v <absolute-path-to-wal-directory>:/etc/agent/data -v <absolute-path-to>/agent.yaml:/etc/agent/agent.yaml -p 12345:12345 grafana/agent
-     ```
+  ...
+}
+```
 
-     > **Note:** On Linux systems, if \<ingress-host\> cannot be resolved by the Grafana Agent, use the additional command-line flag `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` to set it up.
+For more information about the `prometheus.remote_write` component, refer to [prometheus.remote_write](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/prometheus.remote_write) in the Grafana Alloy documentation.
```

Comment:
Is there some magic that replaces `ALLOY_VERSION` as part of the docs build process?

Reply:
Yes. The product version variable is substituted during the build process https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution  The correct version is automatically inserted (unless overrides are set in the front matter of the page).  This keeps links in a specific doc version within that version instead of landing on `latest`.

If this page is backported, then any links go to the correct version of the documentation it references.